taxonomy: add some structure to data quality (#8082)
openfoodfacts/openfoodfacts-server: taxonomies/data_quality.txt

## --------------- Start with data quality errors. See: https://world.openfoodfacts.org/data-quality-errors

# Data quality errors should not contain false positive, as opposed to data quality warnings
en:Data quality errors
description:en: Error detected on data

<en:Data quality errors
en:Nutrition errors

> description:en:Error on nutrition data (nutriments)

Reviewer: ```suggestion
description:en:Error on nutrition data (nutrients)
```

#### Food groups

<Data quality warnings
en:Food groups warnings

> description:en: Food groups can't be infered

Reviewer 2: ```suggestion
description:en: Food groups can't be inferred
```

<en:Food groups warnings
en:Food groups 3 unknown
#description:en:

<en:Food groups warnings
en:Food groups 2 unknown
#description:en:

<en:Food groups warnings
en:Food groups 1 unknown
#description:en:

<en:Food groups warnings
en:Food groups 1 known
#description:en:

<en:Food groups warnings
en:Food groups 2 known
#description:en:

<en:Food groups warnings
en:Food groups 3 known
#description:en:

<en:Data quality ok tags
en:Ingredients percent analysis OK
#description:en:

<en:Data quality warnings
en:Serving quantity defined but quantity undefined
#description:en:
fix_action:en: add_quantity

### Fibers

<en:Data quality warnings

> en:Fiber warnings

Reviewer 3: ```suggestion
en:Fibers warnings
```

> description:en: Their might be an issue with fiber nutriment

Reviewer 3: ```suggestion
description:en: There might be an issue with the fibers nutrient
```

### Carbon footprint

<en:Data quality warnings
en:Carbon footbrint warnings

<en:Carbon footbrint warnings
en:Carbon footprint from known ingredients but not from meat or fish
#description:en:

<en:Carbon footbrint warnings
en:Carbon footprint from known ingredients less than from meat or fish
#description:en:

> <en:Carbon footbrint warnings

Author: :sweat_smile: thanks @diivi !
